HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -1,14 +1,12 @@
+#include "FastTimerService.h"
+
```

Comment:
```suggestion
```

Please include the local headers last.

Reply:
Done. I restored this back the way it was before both here and also in ProcessCallGraph.cc.

I like including the related header first to test it will compile standalone, but I understand this is a debatable style and there are reasons to do it the other way. Since you are the primary maintainer I put them back the way they were.